src/test/resources/files/input/actionCancelInstruction.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<ins:actionInstruction xmlns:ins="http://ons.gov.uk/ctp/response/action/message/instruction">
+    <actionCancel>
+        <actionId>a133b2cd-ef3f-4755-ad07-616b7298a12a</actionId>
+        <responseRequired>false</responseRequired>
+        <actionPlan>ad3fbcb5-3681-4540-b386-ff719ec93643</actionPlan>
+        <actionType>Cancel</actionType>
```

Comment:
This has refreshed my memory about the code we are sending to comet for the cancel forever case. Will need to confirm with Olu if there is a specific code we need to set.